src/DefaultRequestHandlerModel.hpp

```diff
@@ -102,14 +87,14 @@ class DefaultRequestHandlerModel : public RequestHandlerConcept {
 
   void start(const nlohmann::json& /*args*/)
   {
-    //m_run_marker.store(true);
-    m_stats_thread.set_work(&DefaultRequestHandlerModel<RawType>::run_stats, this);
-    m_executor = std::thread(&DefaultRequestHandlerModel<RawType>::executor, this);
+    m_run_marker.store(true);
+    m_stats_thread.set_work(&DefaultRequestHandlerModel<RawType, LatencyBufferType>::run_stats, this);
```

Comment:
I don't remember why there is discrepancy here that we use ReusableThread for one of it and an std::thread for the other. I think where we don't set different works, we don't need ReusableThreads. Can you please switch here to std::thread? Later we can go through the codebase for another iteration. The only place where ReusableThread could be useful in this context is the TriggerMatching mechanism to replace the async calls with setting new work.